vernac/indschemes.ml

```diff
@@ -433,6 +433,8 @@ tried to declare different schemes at once *)
       if not (List.is_empty ischeme) then do_mutual_induction_scheme ischeme
       else
         let mind,l = get_common_underlying_mutual_inductive env escheme in
+        if l <> [] then  (* we don't yet support naming here *)
+          user_err Pp.(str "Naming of identifiers for Scheme Equality not supported.");
```

Comment:
```suggestion
          user_err Pp.(str "Naming of identifiers is not supported for Scheme Equality.");
```

Reply:
I originally wrote this in "better" English, but I didn't want to diverge too much from the quality of the other messages. Otherwise it can look a bit odd. I'd like to hear your thoughts on this however.

Reply:
It might be prudent to create an issue and have a quick pass through all similar error messages (especially in this area of the code) and touch them up a bit at some point.

Reply:
> I originally wrote this in "better" English, but I didn't want to diverge too much from the quality of the other messages.

I realize poor wording is a time-honored tradition for error messages :-), but it's better to raise the bar on quality.  As for improving other messages, I would start with the worst messages first (most unclear or with really poor grammar).  There are so many other things to improve in Coq and its documentation.